.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,48 @@
+name: PR Build Compose
```

Comment:
```suggestion
name: Build pull request
```

Reply:
Also maybe adjust the filename too.

Reply:
Are you sure you don't want "Compose" in the title? I am trying to make the title unique from [the flutter pr build workflow](https://github.com/ReVanced/revanced-manager/blob/dev/.github/workflows/pr-build.yml)

Reply:
Plus the "PR Build" part is to also follow the same format as the flutter workflow

Reply:
Yes, the branch is after all a feature branch, and supposed to eventually reach main. The workflow itself acts without regard to any other for that reason. Besides, "Build pull request" is a better sentencea

Reply:
Sounds good